Documentation/compatibility/wpf-focus-visual-for-radiobutton-and-checkbox-displays-correctly-when-there-is-no-content.md

```diff
@@ -0,0 +1,47 @@
+## WPF FocusVisual for RadioButton and CheckBox Now Displays Correctly When The Controls Have No Content
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In the .NET Framework 4.7.1 and earlier versions, WPF `System.Windows.Controls.CheckBox` and `System.Windows.Controls.RadioButton` have inconsistent and, in Classic and High Contrast themes,
```

Comment:
Here again, links would be good:
- Checkbox: \<xref:System.Windows.Controls.CheckBox?displayProperty=nameWIthType>
- RadioButton: \<xref:System.Windows.Controls.RadioButton?displayProperty=nameWithType>